tools/rpmkeys.c

```diff
@@ -42,6 +41,20 @@ static struct poptOption optionsTable[] = {
     POPT_TABLEEND
 };
 
+static ARGV_t gpgkeyargs(ARGV_const_t args) {
+    ARGV_t gpgargs = argvNew();
```

Comment:
argvNew() is wholly unnecessary here (it almost always is), just initialize to NULl and argvAdd() and similar will alloc it as needed.